2024-05-23 09:40:40.105 - DBeaver 24.0.3.202404211624 is starting
2024-05-23 09:40:40.106 - OS: Windows 10 10.0 (amd64)
2024-05-23 09:40:40.106 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2024-05-23 09:40:40.106 - Install path: 'C:\Users\21928199\AppData\Local\DBeaver'
2024-05-23 09:40:40.107 - Instance path: 'file:/D:/multicampus-workspace/04_servletjsp/itj/'
2024-05-23 09:40:40.107 - Memory available 64Mb/1024Mb
2024-05-23 09:40:40.111 - Create display
2024-05-23 09:40:40.176 - Initialize desktop platform...
2024-05-23 09:40:40.268 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.200.v20231102-0934]
> Start Core Resource Management [org.eclipse.core.resources 3.20.0.v20231102-0934]
> Start 팀 지원 핵심 [org.eclipse.team.core 3.10.200.v20231106-1240]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.129.202404211624]
2024-05-23 09:40:40.416 - Initialize base platform...
2024-05-23 09:40:40.431 - Platform initialized (255ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.129.202404211624]
2024-05-23 09:40:40.433 - No port specified for the instance controller to connect to
2024-05-23 09:40:40.486 - Starting instance server at http://localhost:56569
2024-05-23 09:40:40.495 - Run workbench
> Start Eclipse e4 작업영역 SWT [org.eclipse.e4.ui.workbench.swt 0.17.200.v20231030-2045]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.200.v20231103-2012]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.200.v20231030-2045]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.200.v20231030-2045]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.200.v20231030-2045]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.200.v20231108-1058]
> Start 디버그 핵심 [org.eclipse.debug.core 3.21.200.v20231021-1513]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.36.0.v20231002-1156]
> Start jna [com.sun.jna 5.13.0.v20230812-1000]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.200.v20230913-1724]
> Start Eclipse e4 작업영역 기능 추가 [org.eclipse.e4.ui.workbench.addons.swt 1.5.200.v20231106-1600]
> Start Eclipse e4 작업영역 SWT 렌더러 [org.eclipse.e4.ui.workbench.renderers.swt 0.16.200.v20231030-2045]
> Start 인터넷 연결 관리 사용자 인터페이스 [org.eclipse.ui.net 1.5.200.v20231106-1240]
> Start Internet Connection Management [org.eclipse.core.net 1.5.200.v20231106-1240]
> Start JSch 핵심 [org.eclipse.jsch.core 1.5.200.v20231106-1240]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.100.v20231103-2012]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.200.v20231025-0632]
2024-05-23 09:40:44.221 - Configure workbench window
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.119.202404211624]
2024-05-23 09:40:44.307 - Total database drivers: 112 (112)
> Start 도움말 시스템 코어 [org.eclipse.help 3.10.200.v20231106-1240]
2024-05-23 09:40:44.369 - Create actions
2024-05-23 09:40:44.373 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2024-05-23 09:40:44.373 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2024-05-23 09:40:44.374 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
2024-05-23 09:40:44.374 - Create workbench actions
2024-05-23 09:40:44.608 - Initialize workbench window
2024-05-23 09:40:44.611 - Finish initialization
> Start 기본 문서 편집기 [org.eclipse.ui.editors 3.17.100.v20231106-1600]
> Start 텍스트 편집기 프레임워크 [org.eclipse.ui.workbench.texteditor 3.17.200.v20231106-1600]
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.129.202404211624]
Warning: NLS unused message: actions_navigator__objects in: org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.129.202404211624]
> Start DBeaver SQL Terminal [org.jkiss.dbeaver.ui.editors.sql.terminal 1.0.105.202404211624]
> Start 핵심 비교 지원 [org.eclipse.compare.core 3.8.300.v20231106-1240]
> Start DBeaver Sample Database [org.jkiss.dbeaver.ext.sample.database 1.0.166.202404211624]
> Start DBeaver Tip of the day [org.jkiss.dbeaver.ext.ui.tipoftheday 1.0.135.202404211624]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.14.100.202311222042]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.16.0.202311222042]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.89.202404211624]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.89.202404211624]
> Start DBeaver Dashboard UI [org.jkiss.dbeaver.ui.dashboard 1.0.126.202404211624]
> Start DBeaver Dashboards Model [org.jkiss.dbeaver.model.dashboard 1.0.2.202404211624]
> Start DBeaver JDBC Model [org.jkiss.dbeaver.model.jdbc 1.0.22.202404211624]
> Start DBeaver Oracle Model [org.jkiss.dbeaver.ext.oracle 2.1.214.202404211624]
> Start DBeaver UI Editors - Connections [org.jkiss.dbeaver.ui.editors.connection 1.0.120.202404211624]
> Start DBeaver SQL Language Semantic Model [org.jkiss.dbeaver.model.lsm 1.0.24.202404211624]
> Start 콘솔 [org.eclipse.ui.console 3.13.100.v20230914-0751]
2024-05-23 09:40:46.080 - Open [https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/8.2.0/mysql-connector-j-8.2.0.pom]
2024-05-23 09:40:46.878 - Open [https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/3.21.9/protobuf-java-3.21.9.pom]
2024-05-23 09:40:47.042 - Open [https://repo1.maven.org/maven2/com/google/protobuf/protobuf-parent/3.21.9/protobuf-parent-3.21.9.pom]
2024-05-23 09:40:47.214 - Open [https://repo1.maven.org/maven2/com/google/protobuf/protobuf-bom/3.21.9/protobuf-bom-3.21.9.pom]
2024-05-23 09:40:48.369 - Open [https://dbeaver.io/product/version.xml]
> Start DBeaver Tasks - Native Tools UI [org.jkiss.dbeaver.tasks.native.ui 1.0.107.202404211624]
> Start DBeaver MySQL UI Support [org.jkiss.dbeaver.ext.mysql.ui 1.0.121.202404211624]
> Start DBeaver Tasks - General UI [org.jkiss.dbeaver.tasks.ui 1.0.107.202404211624]
2024-05-23 09:40:53.249 - Open [https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/8.2.0/mysql-connector-j-8.2.0.jar]
2024-05-23 09:40:54.539 - Open [https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/3.21.9/protobuf-java-3.21.9.jar]
2024-05-23 09:40:57.027 - Open [https://www.gnu.org/licenses/old-licenses/lgpl-2.0.txt]
2024-05-23 09:40:57.936 - Connect with 'localhost' (mysql8-18f85355d93-de9e15e653cb405)
> Start DBeaver MySQL Support [org.jkiss.dbeaver.ext.mysql 2.1.214.202404211624]
2024-05-23 09:40:57.958 - Execution context opened (localhost; Main; 0)
> Start DBeaver GIS Support [org.jkiss.dbeaver.data.gis 1.0.153.202404211624]
2024-05-23 09:40:58.045 - Public Key Retrieval is not allowed
org.jkiss.dbeaver.model.exec.DBCConnectException: Public Key Retrieval is not allowed
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.openConnection(MySQLDataSource.java:481)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.<init>(MySQLDataSource.java:121)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.<init>(MySQLDataSource.java:116)
	at org.jkiss.dbeaver.ext.mysql.MySQLDataSourceProvider.openDataSource(MySQLDataSourceProvider.java:154)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLNonTransientConnectionException: Public Key Retrieval is not allowed
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:111)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 16 more
Caused by: com.mysql.cj.exceptions.UnableToConnectException: Public Key Retrieval is not allowed
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:131)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:49)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:446)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:215)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1428)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:133)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:935)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:805)
	... 22 more
2024-05-23 09:41:09.954 - Connect with 'localhost' (mysql8-18f85355d93-de9e15e653cb405)
2024-05-23 09:41:09.954 - Execution context opened (localhost; Main; 1)
2024-05-23 09:41:09.957 - Public Key Retrieval is not allowed
org.jkiss.dbeaver.model.exec.DBCConnectException: Public Key Retrieval is not allowed
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.openConnection(MySQLDataSource.java:481)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.<init>(MySQLDataSource.java:121)
	at org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource.<init>(MySQLDataSource.java:116)
	at org.jkiss.dbeaver.ext.mysql.MySQLDataSourceProvider.openDataSource(MySQLDataSourceProvider.java:154)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:117)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.DBUtils.initDataSource(DBUtils.java:2505)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:91)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:89)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:73)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLNonTransientConnectionException: Public Key Retrieval is not allowed
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:111)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 28 more
Caused by: com.mysql.cj.exceptions.UnableToConnectException: Public Key Retrieval is not allowed
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:131)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:49)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:446)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:215)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1428)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:133)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:935)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:805)
	... 34 more
2024-05-23 09:41:54.743 - Open [https://repo1.maven.org/maven2/com/oracle/database/jdbc/ojdbc11/23.2.0.0/ojdbc11-23.2.0.0.pom]
2024-05-23 09:41:55.175 - Open [https://repo1.maven.org/maven2/com/oracle/database/nls/orai18n/23.2.0.0/orai18n-23.2.0.0.pom]
2024-05-23 09:41:55.320 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xdb6/18.3.0.0/xdb6-18.3.0.0.pom]
2024-05-23 09:41:55.479 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xmlparserv2/18.3.0.0/xmlparserv2-18.3.0.0.pom]
2024-05-23 09:41:55.631 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xmlparserv2/23.2.0.0/xmlparserv2-23.2.0.0.pom]
2024-05-23 09:41:56.919 - Open [https://repo1.maven.org/maven2/com/oracle/database/jdbc/ojdbc11/23.2.0.0/ojdbc11-23.2.0.0.jar]
2024-05-23 09:41:58.527 - Open [https://repo1.maven.org/maven2/com/oracle/database/nls/orai18n/23.2.0.0/orai18n-23.2.0.0.jar]
2024-05-23 09:41:58.817 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xdb6/18.3.0.0/xdb6-18.3.0.0.jar]
2024-05-23 09:41:58.994 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xmlparserv2/18.3.0.0/xmlparserv2-18.3.0.0.jar]
2024-05-23 09:42:01.521 - Open [https://repo1.maven.org/maven2/com/oracle/database/xml/xmlparserv2/23.2.0.0/xmlparserv2-23.2.0.0.jar]
2024-05-23 09:42:01.871 - Connect with 'ORCL' (oracle_thin-18f85335347-28a4660c2894daa5)
2024-05-23 09:42:01.877 - Execution context opened (ORCL; Main; 2)
5월 23, 2024 9:42:01 오전 oracle.jdbc.driver.PhysicalConnection connect
정보: I:thread-37 DBeaver: Loading entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@21d2c80e)
5월 23, 2024 9:42:01 오전 oracle.net.ns.NSProtocol connect
정보: U:thread-37 DBeaver: Loading traceId=5DB18CDE. 
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocol establishConnection
정보: U:thread-37 DBeaver: Loading Session Attributes: 
sdu=8192, tdu=2097152
nt: host=localhost, port=1521, socketOptions={0=YES, 1=NO, 2=0, 38=TLS, 40=false, 12=WINDOWS-ROOT, 17=0, 18=false, 20=true, 23=40, 24=50, 25=0}
    socket=null
client profile={oracle.net.encryption_types_client=(), oracle.net.crypto_seed=, oracle.net.authentication_services=(BEQ), oracle.net.setFIPSMode=false, oracle.net.kerberos5_mutual_authentication=false, oracle.net.encryption_client=ACCEPTED, oracle.net.crypto_checksum_client=ACCEPTED, oracle.net.crypto_checksum_types_client=()}
onBreakReset=false, dataEOF=false, negotiatedOptions=0x0, connected=false
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocol configureSessionAttsAno
정보: U:thread-37 DBeaver: Loading traceId=5DB18CDE, anoEnabled=true. 
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocolNIO handleConnectPacketResponse
정보: U:thread-37 DBeaver: Loading Got Refused, SessionTraceId = 5DB18CDE
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocolNIO establishConnectionAfterRefusePacket
정보: U:thread-37 DBeaver: Loading Outbound interrupt timer cancelled null
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocol establishConnection
정보: U:thread-37 DBeaver: Loading Session Attributes: 
sdu=8192, tdu=2097152
nt: host=localhost, port=1521, socketOptions={0=YES, 1=NO, 2=0, 38=TLS, 40=false, 12=WINDOWS-ROOT, 17=0, 18=false, 20=true, 23=40, 24=50, 25=0}
    socket=null
client profile={oracle.net.encryption_types_client=(), oracle.net.crypto_seed=, oracle.net.authentication_services=(BEQ), oracle.net.setFIPSMode=false, oracle.net.kerberos5_mutual_authentication=false, oracle.net.encryption_client=ACCEPTED, oracle.net.crypto_checksum_client=ACCEPTED, oracle.net.crypto_checksum_types_client=()}
onBreakReset=false, dataEOF=false, negotiatedOptions=0x0, connected=false
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocolNIO handleConnectPacketResponse
정보: U:thread-37 DBeaver: Loading Got Refused, SessionTraceId = 5DB18CDE
5월 23, 2024 9:42:02 오전 oracle.net.ns.NSProtocolNIO establishConnectionAfterRefusePacket
정보: U:thread-37 DBeaver: Loading Outbound interrupt timer cancelled null
5월 23, 2024 9:42:02 오전 oracle.jdbc.driver.PhysicalConnection connect
정보: I:thread-37 DBeaver: Loading throwing
java.sql.SQLRecoverableException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1099)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:879)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:117)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.DBUtils.initDataSource(DBUtils.java:2505)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:91)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:89)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:73)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: oracle.net.ns.NetException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
	at oracle.net.ns.NSProtocolNIO.createRefusePacketException(NSProtocolNIO.java:913)
	at oracle.net.ns.NSProtocolNIO.handleConnectPacketResponse(NSProtocolNIO.java:454)
	at oracle.net.ns.NSProtocolNIO.negotiateConnection(NSProtocolNIO.java:262)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectSession(T4CConnection.java:2882)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:811)
	... 34 more

2024-05-23 09:42:02.040 - ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:117)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.DBUtils.initDataSource(DBUtils.java:2505)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:91)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:89)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:73)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLRecoverableException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1099)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:879)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 27 more
Caused by: oracle.net.ns.NetException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=7EC/YMPQSEqFgGQk0otVvQ==)
	at oracle.net.ns.NSProtocolNIO.createRefusePacketException(NSProtocolNIO.java:913)
	at oracle.net.ns.NSProtocolNIO.handleConnectPacketResponse(NSProtocolNIO.java:454)
	at oracle.net.ns.NSProtocolNIO.negotiateConnection(NSProtocolNIO.java:262)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectSession(T4CConnection.java:2882)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:811)
	... 34 more
> Start DBeaver Oracle UI [org.jkiss.dbeaver.ext.oracle.ui 1.0.116.202404211624]
2024-05-23 09:42:15.055 - Auth model 'native' was replaced by 'oracle_native'
2024-05-23 09:42:15.072 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found
2024-05-23 09:42:20.688 - Connect with 'jdbc:oracle:thin:@//localhost:1521/ORCL' (oracle_thin-18fa2e61c79-70d842a7ed525d15)
2024-05-23 09:42:20.688 - Execution context opened (jdbc:oracle:thin:@//localhost:1521/ORCL; Main; 3)
5월 23, 2024 9:42:19 오전 oracle.jdbc.driver.PhysicalConnection connect
정보: I:thread-63 Connect to 'Oracle' entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@55a90b3e)
5월 23, 2024 9:42:19 오전 oracle.net.ns.NSProtocol connect
정보: U:thread-63 Connect to 'Oracle' traceId=61FEDE6D. 
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocol establishConnection
정보: U:thread-63 Connect to 'Oracle' Session Attributes: 
sdu=8192, tdu=2097152
nt: host=localhost, port=1521, socketOptions={0=YES, 1=NO, 2=0, 38=TLS, 40=false, 12=WINDOWS-ROOT, 17=0, 18=false, 20=true, 23=40, 24=50, 25=0}
    socket=null
client profile={oracle.net.encryption_types_client=(), oracle.net.crypto_seed=, oracle.net.authentication_services=(BEQ), oracle.net.setFIPSMode=false, oracle.net.kerberos5_mutual_authentication=false, oracle.net.encryption_client=ACCEPTED, oracle.net.crypto_checksum_client=ACCEPTED, oracle.net.crypto_checksum_types_client=()}
onBreakReset=false, dataEOF=false, negotiatedOptions=0x0, connected=false
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocol configureSessionAttsAno
정보: U:thread-63 Connect to 'Oracle' traceId=61FEDE6D, anoEnabled=true. 
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocolNIO handleConnectPacketResponse
정보: U:thread-63 Connect to 'Oracle' Got Refused, SessionTraceId = 61FEDE6D
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocolNIO establishConnectionAfterRefusePacket
정보: U:thread-63 Connect to 'Oracle' Outbound interrupt timer cancelled null
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocol establishConnection
정보: U:thread-63 Connect to 'Oracle' Session Attributes: 
sdu=8192, tdu=2097152
nt: host=localhost, port=1521, socketOptions={0=YES, 1=NO, 2=0, 38=TLS, 40=false, 12=WINDOWS-ROOT, 17=0, 18=false, 20=true, 23=40, 24=50, 25=0}
    socket=null
client profile={oracle.net.encryption_types_client=(), oracle.net.crypto_seed=, oracle.net.authentication_services=(BEQ), oracle.net.setFIPSMode=false, oracle.net.kerberos5_mutual_authentication=false, oracle.net.encryption_client=ACCEPTED, oracle.net.crypto_checksum_client=ACCEPTED, oracle.net.crypto_checksum_types_client=()}
onBreakReset=false, dataEOF=false, negotiatedOptions=0x0, connected=false
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocolNIO handleConnectPacketResponse
정보: U:thread-63 Connect to 'Oracle' Got Refused, SessionTraceId = 61FEDE6D
5월 23, 2024 9:42:20 오전 oracle.net.ns.NSProtocolNIO establishConnectionAfterRefusePacket
정보: U:thread-63 Connect to 'Oracle' Outbound interrupt timer cancelled null
5월 23, 2024 9:42:20 오전 oracle.jdbc.driver.PhysicalConnection connect
정보: I:thread-63 Connect to 'Oracle' throwing
java.sql.SQLRecoverableException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1099)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:879)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.runtime.jobs.ConnectionTestJob.run(ConnectionTestJob.java:103)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: oracle.net.ns.NetException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
	at oracle.net.ns.NSProtocolNIO.createRefusePacketException(NSProtocolNIO.java:913)
	at oracle.net.ns.NSProtocolNIO.handleConnectPacketResponse(NSProtocolNIO.java:454)
	at oracle.net.ns.NSProtocolNIO.negotiateConnection(NSProtocolNIO.java:262)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectSession(T4CConnection.java:2882)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:811)
	... 23 more

2024-05-23 09:42:20.751 - ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.runtime.jobs.ConnectionTestJob.run(ConnectionTestJob.java:103)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLRecoverableException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1099)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:879)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 16 more
Caused by: oracle.net.ns.NetException: ORA-12514: 데이터베이스에 접속할 수 없습니다. %s 서비스가 %s의 리스너에 등록되지 않았습니다. (CONNECTION_ID=P0zP7BQsSXe+jIhjO72aTg==)
	at oracle.net.ns.NSProtocolNIO.createRefusePacketException(NSProtocolNIO.java:913)
	at oracle.net.ns.NSProtocolNIO.handleConnectPacketResponse(NSProtocolNIO.java:454)
	at oracle.net.ns.NSProtocolNIO.negotiateConnection(NSProtocolNIO.java:262)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectSession(T4CConnection.java:2882)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:811)
	... 23 more
2024-05-23 09:46:01.027 - DBeaver is stopping
2024-05-23 09:46:01.028 - Stop instance server
2024-05-23 09:46:02.032 - Instance server has been stopped
2024-05-23 09:46:02.033 - DBeaver shutdown completed
